### PR TITLE
Pam 45 frontend employee can add new assets

### PR DESCRIPTION
**Pull request by anantKC**

### CODE REVIEW

File: src/modules/assets/requestedAssets/views/ViewMyAssetList.vue [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+  components: {},
+  async created() {
+    //this block is used for testing
+    try {
+      const response = await axios.get(
+        `http://localhost:3000/my_assets_list`
+      );
+      this.my_assets = response.data;
+    } catch (e) {
+      console.error(e);
```
Comment (asimnp) [inline src/modules/assets/requestedAssets/views/ViewMyAssetList.vue]: Remove console, use alert instead.

File: src/modules/assets/requestedAssets/views/ViewMyAssetList.vue [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+            <th scope="col">S.N</th>
+            <th scope="col">Asset</th>
+            <th scope="col">Type</th>
+            <th scope="col">Requested Date</th>
+            <th scope="col">Bought Date</th>
+            <th scope="col">Requested Amount</th>
+            <th scope="col">Status</th>
+          </tr>
+        </thead>
+        <tbody v-for="(asset, index) in my_assets" :key="index">
```
Comment (asimnp) [inline src/modules/assets/requestedAssets/views/ViewMyAssetList.vue]: Do not use index as a key, use asset id

File: src/modules/assets/views/AddAsset.vue [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+        bought_date:this.bought_date,
+        requested_date:this.requested_date
+      };
+    },
+    async handleSubmit() {
+      
+      try {
+        const response = await axios.post("http://localhost:3000/my_assets_list",this.formData());
+        
+      } catch (err) {
```
Comment (asimnp) [inline src/modules/assets/views/AddAsset.vue:33]: Add alert to show error.

File: src/modules/assets/views/AddAsset.vue [hunk 4]
```diff
@@ -0,0 +1,380 @@
...
+        requested_amount:this.requested_amount,
+        status:this.status,
+        bought_date:this.bought_date,
+        requested_date:this.requested_date
+      };
+    },
+    async handleSubmit() {
+      
+      try {
+        const response = await axios.post("http://localhost:3000/my_assets_list",this.formData());
```
Comment (asimnp) [inline src/modules/assets/views/AddAsset.vue:31]: After requesting the assets list, then what?

Comment (anantKC) [reply src/modules/assets/views/AddAsset.vue:31]: we do nothing, just render the list in frontend

File: src/modules/assets/views/AddAsset.vue (hunk 3 above)
Comment (anantKC) [reply src/modules/assets/views/AddAsset.vue:33]: alert will be added by Saugat later

File: src/modules/assets/requestedAssets/views/ViewMyAssetList.vue (hunk 2 above)
Comment (anantKC) [reply src/modules/assets/requestedAssets/views/ViewMyAssetList.vue]: updated

File: src/modules/assets/views/AddAsset.vue (hunk 4 above)
Comment (anantKC) [reply src/modules/assets/views/AddAsset.vue:31]: we just send the data to backend, sent successfully message will be added by saugat

**Review by asimnp — APPROVED** [review]

**Pull request merged**